v2/pubsub-binary-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/PubsubProtoToBigQuery.java

```diff
@@ -192,11 +190,9 @@ public interface PubSubProtoToBigQueryOptions
         optional = true,
         description = "Preserve Proto Field Names",
         helpText =
-            "Flag to control whether proto field names should be kept or converted to"
-                + " lowerCamelCase. If the table already exists, this should be based on what"
-                + " matches the table's schema. Otherwise, it will determine the column names of"
-                + " the created table. True to preserve proto snake_case. False will convert fields"
-                + " to lowerCamelCase. (Default: false)")
+            "`true` to preserve the original Proto field name in JSON. "
```

Comment:
I think some lines got deleted here that shouldn't be.

Reply:
Public doc https://cloud.google.com/dataflow/docs/guides/templates/provided/pubsub-proto-to-bigquery#:~:text=preserveProtoFieldNames description started with true. Now, I have made it more readable. Hope I understood your comment correctly. If not, please elaborate.